### DISCUSSION TITLE
KERI Implementer's Call 05/16/2024

### DISCUSSION THREAD

**Discussion by pfeairheller** [WebOfTrust/keri#81] [Meetings]
New style Implementers call discussion... 

The Primary contact (or backup) for each repo please comment on this discussion prior to the meeting date listed in the title with your update for the week...

KERIpy - @m00sey  (main), @pfeairheller  (back up)
SignifyPy - @pfeairheller  , @m00sey  (back up)
KERIA - @rodolfomiranda , @2byrds (backup)
Signify-TS - @kentbull , @daniellenskjö (backup)
Cesride - @kentbull , @daidoji (backup)
Kerific - @KorDwarshuis
WOT-Terms - @henkvancann

**Comment by kentbull** [comment]
SignifyTS:
- Ready to merge:
  - Lance's configurable signedFetch [#259](https://github.com/WebOfTrust/signify-ts/pull/259)
  - Daniel's do not include src in NPM package [#261](https://github.com/WebOfTrust/signify-ts/pull/261)
  - Daniel's remove development branch from workflows [#262](https://github.com/WebOfTrust/signify-ts/pull/262)
- A few PRs failing on integration tests:
  - Fergal's work on IPEX single sig [#234](https://github.com/WebOfTrust/signify-ts/pull/234) is in the process of being reviewed.
  - Rodo's work on AID rename and deletion [#241](https://github.com/WebOfTrust/signify-ts/pull/241)
  - Adding integration test for multisig join: [#258](https://github.com/WebOfTrust/signify-ts/pull/258)
- Using Dev version of KERIA breaks tests
  - Daniel's PR [#252](https://github.com/WebOfTrust/signify-ts/pull/252)
- Blocked on NPM publishing key/token 
  - Daniel's NPM package publishing PR [#260](https://github.com/WebOfTrust/signify-ts/pull/260)
- Merge Conflicts:
  -  Provenant da-tychinin's PR on signature-params not serialized [#230](https://github.com/WebOfTrust/signify-ts/issues/230)

- Closeable issues:
  - [#227](https://github.com/WebOfTrust/signify-ts/issues/227) Multisig join never resolves. 
    - Fixed as of KERIA [#236](https://github.com/WebOfTrust/keria/pull/236)

CESRide:
- All PRs are over four three months old. We need a maintainer.
  - Charles's PRs:
    - [#166](https://github.com/WebOfTrust/cesride/pull/166/files) Creder -> SerderACDC
    - [#167](https://github.com/WebOfTrust/cesride/pull/167) **really small** PR to use non-deprecated Rust functions.
    - [#165](https://github.com/WebOfTrust/cesride/pull/167) Remove saider add wasm
    -  Draft [#161](https://github.com/WebOfTrust/cesride/pull/161) Update code tables to CESR 2.0

**Comment by m00sey** [comment]
keripy:

1.1.14 released, minor change to GroupMultisigRotate Doer, removes need for bran and passes in the current Habery in place.
Minor update to multisig join command line adds --group arg and removes hard coded test alias.

On main removed some old helpers generateSigners and randomNonce replaced with real implementations Salter.signers() and Salter.qb64 respectively.

New "three stooges" multisig rotate demo script.

**Comment by rodolfomiranda** [comment]
KERIA:
- Merged PR https://github.com/WebOfTrust/keria/pull/242 
- No open PRs
- No new issues (40 unresolved)

**Reply by iFergal** [reply]
Just opened https://github.com/WebOfTrust/keria/pull/243 - need to update tests

**Reply by lenkan** [reply]
This issue is the one causing the test failing in signify-ts https://github.com/WebOfTrust/keria/issues/230

**Comment by pfeairheller** [comment]
SignifyPy:

- No new PRs
- No new Issues (9 open)

**Comment by pfeairheller** [comment]
@2byrds bringing up https://github.com/WebOfTrust/keria/issues/230

**Comment by pfeairheller** [comment]
WOT-Terms:  (for @henkvancann )

- https://github.com/WebOfTrust/WOT-terms/issues/149
